source/libs/on-new-comments.ts

```diff
@@ -20,7 +20,7 @@ const addListenersOnNewElements = debounce(() => {
 }, {wait: 50});
 
 const setup = () => {
-	const discussion = select('.js-discussion');
+	const discussion = select<HTMLElement>('.js-discussion');
```

Comment:
Used the generic `HTMLElement` as I have no idea what kind of element this is supposed to be. I would guess a `<div />`, maybe `<textarea />`, but not certain.

Reply:
div